libs/utils/src/lib/media/media.firestore.ts

```
@@ -24,3 +24,20 @@ export function createImgRef(ref: Partial<ImgRef> | string = {}): ImgRef {
     ..._ref
   };
 }
+
+export function getImgSize(url: string) {
+  if (url.includes('avatar')) {
+    return { original: 0, xs: 50, md: 100, lg: 300 };
+  } else if (url.includes('logo')) {
+    return { original: 0, xs: 50, md: 100, lg: 300 };
+  } else if (url.includes('poster')) {
+    return { original: 0, xs: 200, md: 400, lg: 600 };
+  } else if (url.includes('banner')) {
+    return { original: 0, xs: 300, md: 600, lg: 1200 };
+  } else if (url.includes('still')) {
+    return { original: 0, xs: 50, md: 100, lg: 200 };
+  } else {
+    console.warn('No bucket directory, exiting function');
```

Here you can throw an error so our error logger can directly show to the dev the error in a snackbar